src/test/scala/chiselTests/FixedPointSpec.scala

```diff
@@ -20,7 +20,7 @@ class FixedPointLiteralSpec extends FlatSpec with Matchers {
 }
 
 class FixedPointFromBitsTester extends BasicTester {
-    val uint = 3.U
+    val uint = 3.U(4.W)
```

Comment:
This width is necessary, but it shouldn't be. This shouldn't block the PR b/c it is a separate problem though